test.css

```diff
@@ -0,0 +1,127 @@
+body, html {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  background-color: black;
+}
+
+.background-layer {
+  position: absolute;
+  top: -110px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-image: url('./hackerWaterfall.png');
+  opacity: 0.2;
+  z-index: 0; /* Ensure the background layer is behind content */
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  height: 100vh;
+}
+
+.grid-cross-item {
+  background-image: url('./redlines.png');
+  background-size: 200% 200%; /* Adjust this property according to your needs */
+  z-index: 1;
+}
+
+.grid-fill-item {
+  background-color: rgba(255, 0, 0, 0.8);
+  background-image: url('./redlines.png');
+  background-size: 200% 200%; /* Adjust this property according to your needs */
+  z-index: 1;
+}
+
+.bottom-left {
+  position: fixed;
+  bottom: 0;
+  left: 25px;
+  padding: 40px;
+  color: #fff;
+}
+
+.fira-mono-regular {
+  font-family: "Fira Mono", monospace;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 0.2;
+  font-size: 14px;
+  color: #737373;
+  z-index: 1;
+}
+
+.logo {
+  width: 100px; /* Adjust width as needed */
+  height: auto; /* Maintain aspect ratio */
+  fill: white; /* Set fill color to white */
+  z-index: 2;
+  position: absolute;
+    bottom: 50%;
+    right: 50%;
+}
+
+.headingStuff {
+  font-family: "Fira Mono", monospace;
+  color: white;
+  z-index: 2;
+  text-align: right;
+  vertical-align: text-top;
+  font-size: 26px;
+  position: relative;
+    bottom: 143px;
+    left: 200%;
+}
+
+.username {
+  font-family: "Fira Mono", monospace;
+  color: rgb(255, 0, 0);
+  z-index: 2;
+  text-align: right;
+  vertical-align: text-top;
+  font-size: 26px;
+  position: relative;
+    top: 173px;
+    right: 283%;
+}
+
+@keyframes fadeInOut {
+  0% {
+    opacity: 0;
+  }
+  49% {
+    opacity: 0;
+  }
+  50% {
+    opacity: 1;
+  }
+  99% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+
+span{
```

Comment:
In a larger project you would use a classname instead of affective spans everywhere with this